cves/kernel/CVE-2020-11565.yml

```diff
@@ -55,7 +55,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: Out of bounds error within linux kernal. Specifically in an empty nodelist for mempolicy.c during mount the creates an out of bounds write that pretty much only effects the systems availability.
```

Comment:
I would consider re-wording a little for better readability, namely "pretty much"